### PR TITLE
PP-4462 Add a provider contract test state for a stripe account existing

### PR DESCRIPTION
**Pull request by stephencdaly**
Add a state for a stripe account existing for use by selfservice contract tests for the GET and PATCH stripe account setup endpoints

### CODE REVIEW

File: src/test/java/uk/gov/pay/connector/pact/TransactionsApiContractTest.java [hunk 1]
```diff
@@ -150,6 +150,16 @@ public void accountWithTransactionsOnMay_5_2018() {
         setUpCharges(2, Long.toString(accountId), ZonedDateTime.of(2018, 5, 14, 1, 1, 1, 1, ZoneId.systemDefault()));
     }
 
+    @State("a stripe gateway account with external id 42 exists in the database")
+    public void stripeAccountExists() {
+        DatabaseFixtures
+                .withDatabaseTestHelper(dbHelper)
+                .aTestAccount()
+                .withAccountId(42)
```
Comment (cobainc0) [inline src/test/java/uk/gov/pay/connector/pact/TransactionsApiContractTest.java]: Can this be a long value instead? e.g. `42L`

**Review by cobainc0 — CHANGES_REQUESTED** [review]
Looks good, just one small thing for consistency.

File: src/test/java/uk/gov/pay/connector/pact/TransactionsApiContractTest.java (hunk 1 above)
Comment (stephencdaly) [reply src/test/java/uk/gov/pay/connector/pact/TransactionsApiContractTest.java]: Thanks! I've updated

**Review by cobainc0 — APPROVED** [review]
👍

**Pull request merged**